### PR TITLE
Jb repeater notes

### PR DESCRIPTION
**Pull request by jbrezmorf**

### CODE REVIEW

File: src/JobPanel/backend/async_repeater.py [hunk 1]
```diff
@@ -542,14 +587,20 @@ def stop(self):
...
+                # TODO: repeater address is not just its ID, isn't it?
+                # We should send just ID of the repeater that.
+                data = "{}\n{}\n".format(self.repeater_address[-1], self.listen_port).encode()
                 s.sendall(data)
             except ConnectionRefusedError:
                 pass
             finally:
+                logging.info("Initial back connection done. Closing thread.")
                 s.close()
+                break # ?? OK
```
Comment (jbrezmorf) [inline src/JobPanel/backend/async_repeater.py]: Move log into Server handle_connect.
Remove break.

File: src/JobPanel/backend/async_repeater.py [hunk 2]
```diff
@@ -195,6 +204,8 @@ def handle_connect(self):
         self.set_terminator(_terminator)
 
         # create request 0 a answer 0
+        assert self.request_id, 0
+        self.request_id += 1
```
Comment (radeksrb) [inline src/JobPanel/backend/async_repeater.py]: Přesunul jsem do konstruktoru.

File: src/JobPanel/backend/async_repeater.py [hunk 3]
```diff
@@ -524,15 +572,12 @@ def get_answers(self, child_id):
...
 
-    def run(self, timeout = None):
-        logging.info("before loop")
-
-        self.loop_thread = threading.Thread(target=asyncore.loop, kwargs = {'timeout':1})
-        logging.info("start")
-        self.loop_thread.start()
-        logging.info("after start")
 
     def stop(self):
```
Comment (radeksrb) [inline src/JobPanel/backend/async_repeater.py]: hotovo

File: src/JobPanel/backend/async_repeater.py (hunk 1 above)
Comment (radeksrb) [inline src/JobPanel/backend/async_repeater.py]: hotovo

**Pull request merged**